package.json

```diff
@@ -0,0 +1,36 @@
+{
+  "name": "3d-tiles-validator-1.1",
```

Comment:
I would rename the package to `3d-tiles-validator`. Better name once we go to publish this on npm.